addon/components/cfb-form-editor/question/default.js

```diff
@@ -0,0 +1,83 @@
+import { getOwner } from "@ember/application";
+import { action, computed } from "@ember/object";
+import RenderComponent from "ember-validated-form/components/validated-input/-themes/uikit/render";
+
+import { TYPE_MAP } from "ember-caluma/lib/field";
+
+export default class CfbFormEditorQuestionDefault extends RenderComponent {
+  @computed("model.{id,__typename}")
+  get question() {
+    const raw = {
+      ...this.model.get("data"),
+      ...this.model.get("change"),
+    };
+
+    if (
+      ["ChoiceQuestion", "MultipleChoiceQuestion"].includes(
+        this.model.__typename
+      )
+    ) {
+      // Use Power Select for choice questions to save space.
+      raw.meta = { widgetOverride: "cf-field/input/powerselect" };
+
+      const key = this.model.__typename
+        .replace(/^./, (match) => match.toLowerCase())
+        .replace("Question", "Options");
+
+      raw[key] = raw.options;
+      delete raw.options;
+    }
+
+    return raw;
+  }
+
+  get field() {
+    // In order to mock the field, we need a document instance and its form.
+    const document = getOwner(this)
+      .factoryFor("caluma-model:document")
+      .create({
+        raw: {
+          id: btoa("Document:dv-document"),
+          rootForm: {
+            slug: "dv-form",
```

Comment:
I was wondering, if there is a good way to get the currently selected form.

There seems to be a way to get to the slug through private properties of the Ember components. But aside from the private properties that might break with every update it would also include a request to the API.

Do you have an idea, how we could access the form from withing `cfb-form-editor/question`?

Reply:
Hm, the only idea I have is passing it down

Reply:
I found a way to access the slug. I realized that the slug is part of the URL and thus accessible via the router service.
`this.router.currentRoute.attributes.formSlug`